app/assets/src/components/ui/controls/buttons/DownloadButton.jsx

```diff
@@ -15,6 +19,7 @@ const DownloadButton = ({ disabled, onClick }) => {
 };
 
 DownloadButton.propTypes = {
+  what: PropTypes.string,
```

Comment:
Can you check with Jenn on this ? 
If we need to set the label, I would probably would go with the DropdownDownloadButton to keep consistency of the UI.

Reply:
ok will go with DropdownDownloadButton